addons/calendar/tests/test_event_notifications.py

```diff
@@ -448,3 +449,46 @@ def search_event():
         with freeze_time('2023-11-15 16:00:00'):
             self.assertEqual(len(search_event()), 3)
         events.unlink()
+
+    def test_calendar_recurring_event_delete_notification(self):
+        """ Check that we can delete recurring event notification and decline events. """
```

Comment:
I don't think this message correctly describes the test, it seems that we are "deleting recurrent event notifications", for someone that is trying to understand the context it will be confusing, can you update it please?